docs/guides/high-availability/single-cluster/introduction.adoc

```diff
@@ -11,6 +11,7 @@ summary="Deploy a single Keycloak cluster, optionally stretched across multiple
 The {project_name} single-cluster setup is targeted at use cases that:
 
 * Deploy to an infrastructure with transparent networking, like for example a single {kubernetes} cluster.
+* Desire all healthy ${project_name} instances to handle user requests.
```

Comment:
@ahus1 I changed this from "all nodes serve traffic" as "node" is overloaded in the context of Kubernetes and ultimately it's user requests being routed to the instances. Happy to revert though if you have strong opinions on this.